src/GraphQL/Types/Composite/AutoRegisteringObjectGraphType.cs

```diff
@@ -119,5 +197,21 @@ protected virtual TypeInformation GetTypeInformation(MemberInfo memberInfo)
             typeInformation.ApplyAttributes();
             return typeInformation;
         }
+
+        /// <summary>
+        /// Analyzes a method argument and returns an instance of <see cref="TypeInformation"/>
+        /// containing information necessary to select a graph type. Nullable reference annotations
+        /// are read, if they exist, as well as the <see cref="RequiredAttribute"/> attribute.
+        /// Then any <see cref="GraphQLAttribute"/> attributes marked on the property are applied.
+        /// <br/><br/>
+        /// Override this method to enforce specific graph types for specific CLR types, or to implement custom
+        /// attributes to change graph type selection behavior.
+        /// </summary>
+        protected virtual TypeInformation GetTypeInformation(ParameterInfo parameterInfo)
```

Comment:
Alas `ParameterInfo` and `MemberInfo` do not have common base class, but they inherit from `ICustomAttributeProvider` so `GetTypeInformation` may be  merged into `GetTypeInformation` above into new type case inside switch.

\+ protected virtual TypeInformation GetTypeInformation(MemberInfo memberInfo) -> protected virtual TypeInformation GetTypeInformation(ICustomAttributeProvider provider)

As you see fit.

Reply:
Well, I could.  I would think that `MemberInfo` and `ParameterInfo` better indicate what the methods would support.  I think I'd prefer to leave it, but if you really think it would be better, consolidating the methods would be fine.

Reply:
I understand, so 50/50 😕 .

Reply:
> 50/50

lol!  Let's just leave it then.

Reply:
ok